server/routes/viewingLicences/handlers/viewCases.ts

```diff
@@ -67,6 +68,18 @@ export default class ViewAndPrintCaseRoutes {
     })
   }
 
+  GET_WITH_EXCLUSIONS = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+    const { caseloadsSelected = [] } = req.session
+    const allPrisons = await this.prisonerService.getPrisons()
+    const activeCaseload = allPrisons.filter(p => p.agencyId === user.activeCaseload)
+    const prisonCaseloadToDisplay = caseloadsSelected.length ? caseloadsSelected : [activeCaseload[0].agencyId]
+
+    const cases = await this.caseloadService.getOmuCaseload(user, prisonCaseloadToDisplay)
+    res.header('Content-Type', 'application/json')
+    res.send(JSON.stringify(cases, null, 4))
```

Comment:
just dumping it out as json - might be a bit tricky rendering as it will currently produce objects at varying stages so will have different structures